test/e2e/mock-cdn/update-mock-cdn-files.js

```diff
@@ -0,0 +1,127 @@
+const util = require('util');
+const { writeFileSync } = require('fs');
+
+const exec = util.promisify(require('node:child_process').exec);
+
+const PPOM_VERSION_URL =
+  'https://static.cx.metamask.io/api/v1/confirmations/ppom/ppom_version.json';
+const PPOM_CONFIG_URL =
+  'https://static.metafi.codefi.network/api/v1/confirmations/ppom/config/0x1/';
+const PPOM_STALE_URL =
+  'https://static.metafi.codefi.network/api/v1/confirmations/ppom/stale/0x1/';
+const PPOM_STALE_DIFF_URL =
+  'https://static.metafi.codefi.network/api/v1/confirmations/ppom/stale_diff/0x1/';
+const MOCK_CDN_FOLDER_URL = 'test/e2e/mock-cdn/';
+
+const CDN_CONFIG_PATH = 'test/e2e/mock-cdn/cdn-config.txt';
+const CDN_STALE_DIFF_PATH = 'test/e2e/mock-cdn/cdn-stale-diff.txt';
+const CDN_STALE_PATH = 'test/e2e/mock-cdn/cdn-stale.txt';
+
+async function getFileVersions() {
+  let ppomVersionData;
+
+  ppomVersionData = await fetch(PPOM_VERSION_URL, {
+    method: 'GET',
+  });
+
+  const ppomVersionDataHeaders = ppomVersionData.headers;
+  ppomVersionData = await ppomVersionData.json();
+
+  const etagVersion = ppomVersionDataHeaders.get('etag');
+  const etagVersionObject = { Etag: etagVersion };
+
+  // updating ppom-version-headers.json file
+  writeFileSync(
+    `${MOCK_CDN_FOLDER_URL}ppom-version-headers.json`,
+    JSON.stringify(etagVersionObject, null, 2),
+  );
+
+  // updating ppom-version.json file
+  writeFileSync(
+    `${MOCK_CDN_FOLDER_URL}ppom-version.json`,
+    JSON.stringify(ppomVersionData, null, 2),
+  );
+
+  const mainnetConfigVersion = ppomVersionData.find(
+    (item) => item.name === 'config' && item.chainId === '0x1',
+  ).version;
+
+  const mainnetStaleVersion = ppomVersionData.find(
+    (item) => item.name === 'stale' && item.chainId === '0x1',
+  ).version;
+
+  const mainnetStaleDiffVersion = ppomVersionData.find(
+    (item) => item.name === 'stale_diff' && item.chainId === '0x1',
+  ).version;
+
+  return {
+    mainnetConfigVersion,
+    mainnetStaleVersion,
+    mainnetStaleDiffVersion,
```

Comment:
we need to get each file version, for later make the corresponding request to the config, stale and stale_files, by passing the version into the URL param